### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Gianni-Pucillo**
Small modification to the contribution process document.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,14 +30,21 @@ https://www.eclipse.org/projects/handbook/#resources-commit
...
-Checkout source code of Explorer for z/OS:
 $ git clone https://github.com/eclipse/che-che4z-explorer-for-zos.git
 
-To build project run:
+Build the project running:
 $ npm install && npm run compile
 
+When you finished type the following:
+* $ git add .
+* $ git commit -m "explanation of your work" -s
```
Comment (shciubrcom) [inline CONTRIBUTING.md]: Issue number also has to be in a commit message.

Comment (Gianni-Pucillo) [reply CONTRIBUTING.md]: Thanks. I will describe it.

Comment (Gianni-Pucillo) [reply CONTRIBUTING.md]: ... soon.

Comment (Gianni-Pucillo) [reply CONTRIBUTING.md]: Did other modification. Please let me know.

Comment (shciubrcom) [reply CONTRIBUTING.md]: https://accounts.eclipse.org/legal/eca/validation/12460

Comment (sergiuilie) [reply CONTRIBUTING.md]: @Gianni-Pucillo   please close the pull request if you have another one open for the same issue

Comment (Gianni-Pucillo) [reply CONTRIBUTING.md]: Yes I close it. Thanks

**Pull request closed without merging**